WebPhapp/client/src/App/pages/Home.js

```diff
@@ -1,19 +1,16 @@
-import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
 
 class Home extends Component {
   render() {
     return (
-    <div className="App">
-      <h1>Project Home</h1>
-      {/* Link to List.js */}
-      <Link to={'./list'}>
-        <button variant="raised">
-            My List
-        </button>
-      </Link>
-    </div>
+      <div className="App">
+        <h1>Project Home</h1>
+        {/* Link to List.js */}
+        <Link to={"./list"}>
```

Comment:
It would be nice to have a button also to the prescription page. Could we do this?

Reply:
I have created a new pull request with a button to the prescription row page.